pytket/binders/circuit/library.cpp

```diff
@@ -190,6 +214,17 @@ void init_library(py::module &m) {
   library_m.def(
       "_ZZPhase_using_CX", &CircPool::ZZPhase_using_CX,
       "Equivalent to ZZPhase, using CX and Rz gates");
+  library_m.def(
+      "_TK2_using_ZZPhase", &CircPool::TK2_using_ZZPhase,
+      "Equivalent to TK2, using 3 ZZPhase gates");
+  library_m.def(
+      "_approx_TK2_using_1xZZPhase", &CircPool::approx_TK2_using_1xZZPhase,
+      "Approximate equivalent to TK2, using 1 ZZPhase gate and single-qubit "
+      "gates. Only requires the first angle of the TK2 gate.");
```

Comment:
I think it is still worth saying that all TK2 angles should be normalized -- just descriptively, without reference to the predicate. (Same below.)

Reply:
Actually, for the decompositions using ZZPhase, it works no matter what the normalisation is.

Reply:
Ah OK